php/blocks/class-loader.php

```diff
@@ -150,22 +149,6 @@ protected function editor_assets() {
 			'before'
 		);
 
-		// Enqueue optional editor only styles.
-		wp_enqueue_style(
-			'block-lab-editor-css',
-			$this->assets['url']['editor_style'],
-			[],
-			$this->plugin->get_version()
-		);
-
-		$block_names = wp_list_pluck( $this->blocks, 'name' );
-
-		foreach ( $block_names as $block_name ) {
-			$this->enqueue_block_styles( $block_name, [ 'preview', 'block' ] );
-		}
-
-		$this->enqueue_global_styles();
-
```

Comment:
This is simply moved below, so the script-related logic is first, then the style logic. There's no change to it.